.ci_support/linux_64_.yaml

```diff
@@ -1,15 +1,15 @@
 CHOST:
-- x86_64-pc-windows-msvc19.14.26423
+- x86_64-pc-windows-msvc19.29.30139
 FINAL_CFLAGS:
 - -march=k8 -mtune=haswell -ftree-vectorize -fstack-protector-strong -O2 -ffunction-sections
   -pipe
 FINAL_CXXFLAGS:
 - -fvisibility-inlines-hidden -std=c++14 -fmessage-length=0 -march=k8 -mtune=haswell
   -ftree-vectorize -fstack-protector-strong -O2 -ffunction-sections -pipe
 MSVC_HEADERS_VERSION:
-- 14.16.27023
+- 14.29.30139
```

Comment:
CI is probably failing as the patch version here is different. MS only aligns major and minor with the compiler. Matching compiler and headers will have different patch versions.

Reply:
Ah, I see there are a bunch more variables being set in `activate-msvc-headers-libs.sh` that now have to be updated as well (and should probably move to the cbc)...